src/controls.tsx

```diff
@@ -0,0 +1,140 @@
+// @ts-ignore: library file import
+import { Container, BooleanInput, Label, SliderInput, SelectInput, Panel, Button } from 'lib/pcui/component.js';
+// @ts-ignore: library file import
+import { BindingTwoWay, useObserverState } from 'lib/pcui/binding.js';
+import React, { useEffect, useState, useContext } from 'react';
+import { ProgressPlugin } from 'webpack';
+import { Morph, Option, Observer } from './types';
+
+const ObserverContext = React.createContext(null);
```

Comment:
this looks like an instance of something, in which case shouldn't it be named `observerContext` ? Line below too.

Reply:
In '.jsx / .tsx' files react components are generally pascal case to differentiate them from standard DOM elements when used in markup. If you then create an instance of it that component, that'd usually be camel case.

https://airbnb.io/javascript/react/#naming

Reply:
gotcha, thanks